docs/source/science/HLS/HLSL30.ipynb

```diff
@@ -2,23 +2,87 @@
  "cells": [
```

Comment:
<p>I think this object is unused throughout the notebook (we replace it later without having used it). </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/219/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
<p>but it's not used until it's renewed ? </p>

Reply:
<p>No, that's sort of what my next comment below is about. This AWSSession should be used in write_local_data_and_catalog_s3</p>

Reply:
To be clear the point of this PR is to update the notebook to follow style guidelines and ensure it still works. Debates over code changes should be a new ticket/PR.